### ISSUE TITLE
Update Steam Library Covers

### ISSUE THREAD

**Issue by bibbybabby** [studio-minus/ppg-bugs-and-requests#1894]
**Feature description**
every time i see the vile imagery i get stage 6 breast cancer and violently throw up pure gastric acid, im literally shaking and crying rn while typing this request out. please change the dreaded pngs to the new and shiny ones like seriously why does it show a bruise from an axe????

**Feature purpose**
Consistency

**Additional context**
![image](https://user-images.githubusercontent.com/104321411/181935265-de88e1b0-9132-4072-a4f8-67e98f8a2878.png)
![image](https://user-images.githubusercontent.com/104321411/181935439-e5b03783-1136-44a8-9f35-147aa114739d.png)

**Comment by DindinYT37** [comment]
> i get stage 6 breast cancer and violently throw up pure gastric acid, im literally shaking and crying rn

What is wrong with you?

**Comment by bibbybabby** [comment]
> > i get stage 6 breast cancer and violently throw up pure gastric acid, im literally shaking and crying rn
> 
> What is wrong with you?

everything

**Comment by DindinYT37** [comment]
> > > i get stage 6 breast cancer and violently throw up pure gastric acid, im literally shaking and crying rn
> > 
> > 
> > What is wrong with you?
> 
> everything

I see.

**Comment by PizzaBroadcast** [comment]
> > > > i get stage 6 breast cancer and violently throw up pure gastric acid, im literally shaking and crying rn
> > > 
> > > 
> > > What is wrong with you?
> > 
> > 
> > everything
> 
> I see.

hehe im not apart of this conversation i just like replying to long chains of conversations

**Comment by DindinYT37** [comment]
> hehe im not apart of this conversation i just like replying to long chains of conversations

Stop cluttering the comments.

**Comment by MrHushPPG** [comment]
> > i get stage 6 breast cancer and violently throw up pure gastric acid, im literally shaking and crying rn
> 
> What is wrong with you?



> > > > i get stage 6 breast cancer and violently throw up pure gastric acid, im literally shaking and crying rn
> > > 
> > > 
> > > What is wrong with you?
> > 
> > 
> > everything
> 
> I see.

No, seriously, not everyone wants to see blood on the splash screen of a game. Actually my parents might not like it ahahahahah

**Comment by DindinYT37** [comment]
> No, seriously, not everyone wants to see blood on the splash screen of a game. Actually my parents might not like it ahahahahah

If that already concerns them then how did you get the game?
